lib/engine/ability/README.md

```diff
@@ -193,6 +193,16 @@ This company comes with a share of a corporation when acquired.
   randomly selected at game setup.
 - `corporations`: A list of corporations to be used with `"share": "random_share"`
 
+## stubs
```

Comment:
does this not exist for other games?

Reply:
No. Other games do use stubs, but those are all included in the map definition. The difference with 1858 is that these stubs are not permanent, and the requirement to lay track linking to the specific edge (or edges) of a hex goes away when a private company closes. So I've implemented these as abilities to allow them to be removed at the appropriate time.

Reply:
I think it would be simpler to just make the stubs part of the map like in other games, and when the company is closed/bought, you can delete the stubs from the tile.

Reply:
can you delete all changes you added for this feature?

Reply:
I think it would be simpler and better to just make the stubs part of the map like in other games, and when the company is closed/bought, you can delete the stubs from the tile.

Reply:
The stubs ability has been removed.